terraform/modules/data-share-service/ssm.tf

```diff
@@ -9,3 +9,27 @@ resource "aws_ssm_parameter" "lev_api_client_user" {
   type  = "SecureString"
   value = "gdx-data-share-user"
 }
+
+resource "aws_ssm_parameter" "prisoner_search_api_client_id" {
+  name  = "${var.environment}_prisoner_search_api_client_id"
+  type  = "String"
+  value = "gdx-data-share-poc"
+}
+
+resource "aws_ssm_parameter" "prisoner_search_api_client_secret" {
+  name  = "${var.environment}_prisoner_search_api_client_secret"
+  type  = "SecureString"
+  value = "dummyvalue"
+}
+
+resource "aws_ssm_parameter" "prisoner_event_queue_name" {
+  name  = "${var.environment}_prisoner_event_queue_name"
+  type  = "String"
+  value = "dummyvalue"
```

Comment:
do you want to preseed this with the correct value?

Reply:
Yes probably should - what happens if we over write manually does terraform ever update again?